packages/kit/src/views/ReferFriends/pages/RewardDistributionHistory/index.tsx

```
@@ -64,7 +64,7 @@ const formatSections = (items: IInvitePaidHistory['items']) => {
 export default function RewardDistributionHistory() {
   const originalData = useRef<IInvitePaidHistory['items']>([]);
   const [isLoading, setIsLoading] = useState(false);
-  const [sections, setSections] = useState<
+  let [sections, setSections] = useState<
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'sections' is never reassigned.

'setSections' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)